src/ontology/cl-edit.owl

```diff
@@ -28265,6 +28257,82 @@ AnnotationAssertion(rdfs:label obo:CL_4023081 "inverted L6 intratelencephalic pr
 SubClassOf(obo:CL_4023081 obo:CL_4023050)
 SubClassOf(obo:CL_4023081 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070021))
 
+# Class: obo:CL_4023083 (chandelier cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15378039") obo:IAO_0000115 obo:CL_4023083 "A type of interneuron that are axon-targeting. They can be multipolar or bitufted. Their local axonal clusters are formed by high-frequency branching at shallow angles, often ramifying around, above or below their somata with a high bouton density. The characteristic terminal portions of the axon form short vertical rows of boutons, resembling a chandelier.")
```

Comment:
"A type of" is redundant for classes. Should be "An interneuron that is axon-targeting." 

Also, is any interneuron that is axon-targeting a chandelier cell? I thought that ChCs synapsed to axons of pyramidal cells. Perhaps that information is out-of-scope for now.

Reply:
that's a good point - will try to reorder it a bit so the main part comes first. They definitely do synapse to pyramidal cells - should have been worked it, sorry about that. 

Thinking this instead:

An interneuron that selectively innervates the axon initial segment of pyramidal cells. Their local axonal clusters are formed by high-frequency branching at shallow angles, often ramifying around, above or below their somata with a high bouton density. The characteristic terminal portions of the axon form short vertical rows of boutons, resembling a chandelier. Chandelier cells can be multipolar or bitufted.

Reply:
Awesome!

Reply:
Yep - second version *much* better.